### ISSUE TITLE
Feature Request: Support keyboard shortcut to toggle enable state

### ISSUE THREAD

**Issue by brendonjkding** [Robbendebiene/Gesturefy#606]
Would be useful to temporarily disable and re-enable the gesture.

**Comment by Robbendebiene** [comment]
There already is a "deactivation key" which can be enable in the settings. This will deactivate Gestures **while it's pressed/hold down**. Does this fit your needs?

**Comment by brendonjkding** [comment]
I noticed that. But it would be troublesome if i need to press many other keys in the meantime in a short period of time since I need to hold down it all the time.

**Comment by Robbendebiene** [comment]
Can you give me an example where you want to disable Gesturefy? I feel like it's either a site (like a 3D application) where one uses the right mouse button to navigate, in which case Gesturefy could be disabled for this individual page via the "Exclusions" or one wants to quickly disable Gesturefy because there is another sparely used action on the right mouse button.

> But it would be troublesome if i need to press many other keys in the meantime in a short period of time since I need to hold down it all the time.

Just out of curiosity, when do you need hold down the right mouse button and additionally press some keys (other than the modifier keys) on the keyboard?

**Comment by brendonjkding** [comment]
Yes, the former one is partially right. I usually open a live streaming page, put it in the background and leave the sould open when I am browsing other pages. If I hear something interesting I would switch back to it, watch for a while and then switch again to other pages. All the switching are using Gesturefy.
The problem is that sometimes there will be a game on the the live streaming page which need player to hold W/A/S/D/Shift and to move the right mouse button while holding it. 
However, I can't simply disable this website using "Exclusions" since it's a normal page at other times when I also need to back or forward using Gesturefy.

**Comment by Robbendebiene** [comment]
My only fear is that people might disable Gesturefy without noticing it, but I'll look into it.

**Comment by brendonjkding** [comment]
Thank you. I think that can be avoided by not providing default shortcut.

**Comment by Robbendebiene** [comment]
I agree. My concern was that the webextension "Keyboard Shortcut API" does not allow empty shortcut values. However it turns out it does :)

I'll try to add this feature in the next release..
